project_event/wizard/project_event_wizard.py

```diff
@@ -0,0 +1,220 @@
+            'notes': self.event_notes,
+            'project_type': 'event',
+        }
+        event = self.env['project.project'].create(event_vals)
+        for act in self.activity_line_ids:
+            tasks = self.env['task.template.line.wiz'].search([
+                ('event_wiz_id', '=', self.id),
+                ('activity_wiz_id', '=', act.id),
+            ])
+            activity_vals = {
+                'name': act.name,
+                'project_id': event.id,
+                'responsible_id': act.activity_resp_id.id,
+                'activity_category_id': act.activity_category_id.id,
+                'activity_task_type': 'activity',
+                # add room_id
+                'notes': act.notes,
+                'child_ids': [
+                    (0,
+                     0,
+                     {
+                         'name': task.task_name,
+                         'activity_task_type': 'task',
+                         'task_responsible_id': task.task_resp_id.id,
+                         'task_category_id': task.task_category_id.id,
+                         'department_id': task.department_id.id,
+                         'notes': task.notes,
+                         # add resource_id
+                     }
+                     ) for task in tasks],
+            }
+            activity = self.env['project.task'].create(activity_vals)
+
+    @api.onchange('template_id')
+    def _onchange_template_id(self):
+        template = self.env['event.template'].browse(self.template_id.id)
+        self.name = template.name
+        self.event_resp_id = template.temp_resp_id
+        self.event_notes = template.notes
+
+
+class ActivityTemplateLineWiz(models.TransientModel):
+    _name = "activity.template.line.wiz"
+    _description = "Activity Template Line Wizard"
+
+    event_wiz_id = fields.Many2one(
+        'project.event.wizard',
+        string='Activity Template',
+    )
+    template_id = fields.Many2one(
+        'activity.template',
+        string='Activity Template',
+    )
+    name = fields.Char(
+        string='Activity Title',
+    )
+    activity_resp_id = fields.Many2one(
+        'res.partner',
+        string='Responsible',
+    )
+    activity_category_id = fields.Many2one(
+        'activity.category',
+        string='Category',
+    )
+    room_id = fields.Many2one(
+        'resource.resource',
+        string='Room',
+    )
+    notes = fields.Text(string='Notes')
+
+
+class TaskTemplateLineWiz(models.TransientModel):
+    _name = "task.template.line.wiz"
+    _description = "Task Template Line Wizard"
+
+    event_wiz_id = fields.Many2one(
+        'project.event.wizard',
+        string='Event Wizard',
+    )
+    activity_wiz_id = fields.Many2one(
+        'activity.template.line.wiz',
+        string='Activity wizard',
+    )
+    template_id = fields.Many2one(
+        'task.template',
+        string='Task Template',
+    )
+    task_name = fields.Char(
```

Comment:
@rimbendhaou: Why change to  task_name instead of name. (Ignore if not applicable)